sct-commons/src/test/java/org/lfenergy/compas/sct/commons/dto/ResumedDataTemplateTest.java

```diff
@@ -148,12 +154,64 @@ void testCopyFrom() {
         assertThrows(IllegalArgumentException.class, () -> rDtt_t.setFc(TFCEnum.BL));
     }
 
+    /**
+     * ListFcEnum = CF, DC, SG, SP, ST , SE
+     * Fc value should be in ListFcEnum
+     */
     @Test
-    void testIsUpdatable(){
+    void isUpdatable_shouldReturnTrue_whenFcIsInListFcEnum(){
+        //Given
         ResumedDataTemplate rDtt = DTO.createRTT("pre","lnclass","1");
+        //When  Then
         assertTrue(rDtt.isUpdatable());
+    }
 
+    @Test
+    void isUpdatable_shouldReturnFalse_whenFcIsNotInListFcEnum(){
+        //Given
+        ResumedDataTemplate rDtt = DTO.createRTT("pre","lnclass","1");
         rDtt.getDaName().setFc(TFCEnum.BL);
+        //When Then
         assertFalse(rDtt.isUpdatable());
     }
+
+    @ParameterizedTest
```

Comment:
why not to use a ParameterizedTest for all the "true" results an one for the "false" results ?

Reply:
ok